### PR TITLE
use avatar endpoint for app bar image

### PR DESCRIPTION
**Pull request by learykara**
needs to follow https://github.com/Thinkful/pysplash/pull/2345

### CODE REVIEW

File: src/AppBar/Navigation.jsx [hunk 1]
```diff
@@ -126,7 +126,11 @@ class AppNav extends React.Component {
                     <Notifications />
                     <a className="app-nav-link app-nav-link__toggle" onClick={this._toggleMenu}>
                         <span alt="Menu" className="app-nav-burger"></span>
-                        <Gravatar className="app-nav-gravatar" email={user.tf_login} size={120}/>
+                        <Gravatar
+                            className="app-nav-gravatar"
+                            email=""
+                            src={`${config.api.url}/api/hupers/me/avatar`}
```
Comment (tholex) [inline src/AppBar/Navigation.jsx]: why use the Gravatar tag if it's not fetching a gravatar? `img` should do.

Comment (tholex) [inline src/AppBar/Navigation.jsx]: the class is what determines placement/size etc

Comment (learykara) [inline src/AppBar/Navigation.jsx]: there are already a bunch of places throughout the app where we use the `Gravatar` component even though we use an explicit `src` and not email -- figured if nothing else it makes it easier to quickly find all of the places where we show peoples avatars as opposed to other images. seem a good enough reason to keep it?

Comment (tholex) [inline src/AppBar/Navigation.jsx]: weird that `api.url` comes without `/api` but it's correct :+1:

**Comment by tholex** [comment]
I guess that's alright – np :+1:

**Pull request merged**